src/containers/navigation/navigationTreeId.gql

```diff
@@ -0,0 +1,5 @@
+query navigationTreeIdQuery {
+  primaryShop {
+    defaultNavigationTreeId
+  }
```

Comment:
Currently `withShop` HOC does `primaryShopId` query followed by `shopQuery`, and now this adds a third query. But actually they can all be combined into one. We added `primaryShop` query in the API for this purpose, but never updated the client to use it.

So:
- Merge `shop.gql`, `primaryShopId.gql` and `navigationTreeId.gql` files into a single `primaryShop.gql` file, which should look like this one but with all of the fields from `shopQuery` added.
- Query `defaultNavigationTree` rather than `defaultNavigationTreeId` on `primaryShop`, and move all of the fields from `navigationQuery` into there. You then do not need `navigationQuery`.
- Update `withShop` to do just one query, `primaryShop`.
- Now you should no longer need `getNavigationTree` function. Instead, in `App`, you can get it from `shop.defaultNavigationTree` because it already uses `withShop`.